snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/versioning/SnomedVersioningApiTest.java

```diff
@@ -54,5 +54,12 @@ public void createRegularVersion() {
 	public void createVersionWithSameNameAsBranch() {
 		createVersion(SNOMED_SHORT_NAME, "SnomedVersioningApiTest", getNextAvailableEffectiveDateAsString(SNOMED_SHORT_NAME)).statusCode(409);
 	}
+	
+	@Test
+	public void createForcedVersionWithSameNameAsBranch() {
+		createVersion(SNOMED_SHORT_NAME, "SnomedVersioningApiTest", getNextAvailableEffectiveDateAsString(SNOMED_SHORT_NAME), true).statusCode(201);
```

Comment:
It would be better to create an isolated test case and not one that depends on another. Test cases should not rely on each other.